backend/.env

```
@@ -1,3 +1,4 @@
 DB_URL = mongodb+srv://jamessteppingstone:TijO3Hu5QbkLAJ0W@cluster0.vvaox.mongodb.net/
 PORT = 3000
-ENC_KEY = studyhive123
\ No newline at end of file
+ENC_KEY = studyhive123
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)